stdlib.ispc

```diff
@@ -4207,6 +4423,8 @@ __declspec(safe) static inline uniform float half_to_float(uniform unsigned int1
     }
 }
 
+__declspec(safe) static inline uniform float float16_to_float(uniform float16 hf) { return half_to_float(intbits(hf)); }
```

Comment:
Please rename `float16_to_float` to `half_to_float`. They will be overloaded by input parameter.
Same for `float_to_float16`.

Reply:
> Please add description of new stdlib functions to docs/ispc.rst

I don't really understand the point of having such functions available to users at all. The language has implicit and explicit type conversions which we bend to utilize these functions. I would better deprecate `half_to_float` and `float_to_half` and after some time completely delete them.

> Please rename `float16_to_float` to `half_to_float`. They will be overloaded by input parameter. Same for `float_to_float16`.

I don't think this is a good idea, because it will probably break logic of lookuping needed function.